syft/grid/utils/autoscale/gcloud.py

```diff
@@ -34,23 +35,145 @@ def __init__(self, credentials, project_id, region):
         else:
             terraform_script.init()
 
-    def compute_instance(self, name, machine_type, zone, image_family, apply=True):
+    def expose_port(self, name, ports=None, apply=True):
+        """
+        args:
+            name: name of the resource
+            ports: list of ports to be exposed, defaults to 80
+            apply: to call terraform apply at the end
+        """
+        if not ports:
+            ports = [80]
+
```

Comment:
Why not use port as a default parameter value? Like

```python
def expose_port(self, name, ports=[80], apply=True):
    ...
```
Also, if you want to keep the current way we could do:
```python
if ports is not None:
    ports = [80]
```